src/main/java/com/google/sps/servlets/YoutubeServlet.java

```diff
@@ -42,9 +43,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws ServletException {
     try {
       String url = request.getParameter(URL_PARAMETER);
-      // TODO: add numComments parameter. Comment count will be hard-coded until then.
+      String numCommentsInput = request.getParameter(NUM_COMMENTS_PARAMETER);
+      // If nothing is inputted, length of list is default, otherwise it's what was inputted.
```

Comment:
Haha I definitely had it like this before, but Cis told me to change it. Her reasoning is that since the numComments is always derived from a slider, its default value will be 0. However, having a null check seems more secure, so I think I will go with it.